include/extractor/guidance/intersection_handler.hpp

```diff
@@ -327,6 +327,42 @@ std::size_t IntersectionHandler::findObviousTurn(const EdgeID via_edge,
         return false;
     }();
 
+    // check whether we turn onto a oneway through street. These typically happen at the end of
```

Comment:
This is getting a bit out of hand with all the inline lambdas etc.

Do you think we could refactor the intersection handler a bit to make it easier to follow the logic here?